db/schema.rb

```diff
@@ -0,0 +1,69 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_220_810_124_313) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'authorid'
+    t.integer 'postid'
+    t.text 'text'
+    t.bigint 'author_id', null: false
+    t.bigint 'post_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.integer 'authorid'
+    t.integer 'postid'
+    t.bigint 'author_id', null: false
+    t.bigint 'post_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.integer 'authorid'
```

Comment:
- [ ] In the posts table we already have `author_id`, kindly remove this one here as it unnecessary.

Reply:
Hi @julie-ify Thanks for you feedback, the issues highlighted have now been worked on